firebase_admin/_utils.py

```diff
@@ -14,13 +14,19 @@
 
 """Internal utilities common to all modules."""
 
+import json
+import socket
+
+import googleapiclient
+import httplib2
 import requests
+import six
 
 import firebase_admin
 from firebase_admin import exceptions
 
 
-_STATUS_TO_EXCEPTION_TYPE = {
+_ERROR_CODE_TO_EXCEPTION_TYPE = {
```

Comment:
It seems unwise to mix http status codes with grpc status codes in one map.

And you could always do `_ERROR_CODE_TO_EXCEPTION_TYPE[_HTTP_STATUS_TO_ERROR_CODE[X]]` anyway.

Reply:
Done. Removed the `HTTP status -> Exception type` mappings from this dict, and refactored the code accordingly.